src/components/select/select.spec.js

```diff
@@ -1486,6 +1506,29 @@ describe('<md-select>', function() {
     });
   });
 
+  // Only test with custom components on AngularJS 1.5+
+  if (angular.version.major === 1 && angular.version.minor >= 5) {
+    describe('with custom components', function() {
+
+      if (angular.version.major === 1 && angular.version.minor >= 7) {
+        // The $flushPendingTasks API is new in AngularJS 1.7 and is required to reproduce the issue
```

Comment:
FWIW, `$flushPendingTasks` is an alias for `$timeout#flush()` (which works the same on all 1.5+ versions).